istcover.sty

```diff
@@ -125,10 +125,10 @@
 %%
 %% \makeback
 %% Back page of the cover
-%% 
+%%
 \def\makeback{
 	\clearpage
-	\newgeometry{a4paper,top=2.50cm,left=2.50cm,right=2.50cm,bottom=3.30cm,headsep=0.50cm,footskip=-0.77cm}	
+	\newgeometry{a4paper,top=2.50cm,left=2.50cm,right=2.50cm,bottom=3.30cm,headsep=0.50cm}
```

Comment:
the option `a4paper` raise the following warning (one more :wink:), I think it should safely be removed